unified-codes/apps/web/src/reducers/ExplorerReducer.ts

```diff
@@ -18,6 +18,22 @@ const initialState = (): IExplorerState => {
       rowsPerPage: 25,
       type: 'medicinal_product',
```

Comment:
Thinking from here is to move all the old `Explorer` state to the nested state in `searchBar`, `table` and `toggleBar`. Can then have separate reducers for each of those (`reducers/explorer/searchBar.ts` etc.). Sagas will be easy enough to migrate over, just a case of updating the identifiers of the actions they receive/dispatch!

Since the state is shared, we probably don't want to make them all slices (otherwise the `table` reducer won't be able to see `searchBar.input` and `toggleBar.enabledTypes` when it's filtering data etc.). I think it's OK to just `combineReducers` at the root level... it's a compromise between performance and simplicity, but I think it's about right? Just need to make sure that we avoid the temptation to nest more than 2 levels deep, e.g. that `searchBar`, `table` etc. don't contain any nested state themselves!